client/src/main/scala/drt/client/components/FlightChartComponent.scala

```diff
@@ -57,9 +58,11 @@ object FlightChartComponent {
       else
         200 + widthFactor * props.manifestSummary.nationalities.size
 
+      val isBeforeAgeEligibilityChangeDate : Long => Boolean = scheduled => scheduled < SDate("2023-07-25T00:00").millisSinceEpoch
```

Comment:
26th right?

Reply:
It is 25th only . 26th date mentioned in other test is after eligibility date change.